app/uk/gov/hmrc/plasticpackagingtax/registration/forms/OrganisationType.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+import play.api.data.Form
+import play.api.data.Forms.{mapping, text}
+import play.api.libs.json.{Reads, Writes}
+import uk.gov.hmrc.plasticpackagingtax.registration.forms
+
+object OrgType extends Enumeration {
+  val UK_COMPANY: forms.OrgType.Value                   = Value("UK company")
+  val SOLE_TRADER: forms.OrgType.Value                  = Value("Sole trader")
+  val PARTNERSHIP: forms.OrgType.Value                  = Value("Partnership")
+  val CHARITY_OR_NOT_FOR_PROFIT: forms.OrgType.Value    = Value("Charity or Not for Profit")
+  def withNameOpt(name: String): Option[Value]          = values.find(_.toString == name)
+  implicit val orgTypeReads: Reads[forms.OrgType.Value] = Reads.enumNameReads(OrgType)
+  implicit val orgTypeWrites: Writes[Value]             = Writes.enumNameWrites
+}
+
+case class OrganisationType(answer: OrgType.Value)
+
+object OrganisationType extends CommonFormValidators {
+  lazy val emptyError = "organisationDetails.type.empty.error"
+
+  def form(): Form[OrganisationType] =
+    Form(
+      mapping(
+        "answer" -> text()
+          .verifying(emptyError, contains(OrgType.values.toSeq.map(_.toString)))
+      )(OrganisationType.apply)(OrganisationType.unapply)
+    )
+
+  def apply(value: String): OrganisationType = OrganisationType(OrgType.withNameOpt(value).orNull)
```

Comment:
`Null` in Scala is a smell.

This was my approach https://github.com/hmrc/plastic-packaging-tax-registration-frontend/pull/67/files#diff-eb502f07a3e7e1fefe9fec6789846c27d34ef8a77ddcc1176b960c66b0c63313R21-R28

Reply:
done